src/common/execution.py

```diff
@@ -104,8 +105,24 @@ async def get_last_rewards_update() -> RewardVoteInfo | None:
 
 
 async def get_max_fee_per_gas() -> Wei:
-    priority_fee = await execution_client.eth.max_priority_fee  # type: ignore
+    priority_fee = await calculate_median_priority_fee()  # type: ignore
```

Comment:
I see a lot of queries to execution node inside  `calculate_median_priority_fee`. Maybe try calling `max_priority_fee` as it was before. Then in the case of failure try more complicated way.